agents-common/src/main/java/org/apache/ranger/plugin/service/RangerBasePlugin.java

```diff
@@ -494,6 +497,9 @@ public void cleanup() {
 		if (policyEngine != null) {
 			((RangerPolicyEngineImpl) policyEngine).releaseResources(true);
 		}
+		if (pluginAnalytics!=null) {
```

Comment:
do we need null check ?.

Reply:
I was adding it as a safety net based on other surrounding variables such as policyEnginer. Not really required. Removed it on second thoughts since it is never set to null unlike policyEngine.